quarkchain/cluster/tests/test_shard_state.py

```diff
@@ -80,6 +80,74 @@ def test_shard_state_simple(self):
             {self.genesis_token: 2500000000000000000},
         )
 
+    def test_get_total_balance(self):
+        id_list = [Identity.create_random_identity() for _ in range(3)]
+        acc_list = [Address.create_from_identity(i, full_shard_key=0) for i in id_list]
+        env = get_test_env(
+            genesis_account=acc_list[0], genesis_minor_quarkash=100000000,
+        )
+
+        qkc_token = token_id_encode("QKC")
+
+        state = create_default_shard_state(env=env)
+        # Add a root block to have all the shards initialized
+        root_block = state.root_tip.create_block_to_append().finalize()
+        state.add_root_block(root_block)
+        tx1 = create_transfer_transaction(
+            shard_state=state,
+            key=id_list[0].get_key(),
+            from_address=acc_list[0],
+            to_address=acc_list[1],
+            value=12345,
+            gas=50000,
+            transfer_token_id=qkc_token,
+            gas_price=0,
+        )
+        self.assertTrue(state.add_tx(tx1))
+        b1 = state.create_block_to_mine(address=acc_list[2])
+        state.finalize_and_add_block(b1)
+
+        tx2 = create_transfer_transaction(
```

Comment:
what's the point of the second transaction?

Reply:
oh... just want to make it sure that it works in complicated situation at the first time, will remove.